app/Http/Controllers/Api/UsersController.php

```diff
@@ -67,7 +67,7 @@ public function index(Request $request)
             'users.remote',
             'users.ldap_import',
 
-        ])->with('manager', 'groups', 'userloc', 'company', 'department', 'assets', 'licenses', 'accessories', 'consumables', 'adminuser',)
+        ])->with('manager', 'groups', 'userloc', 'company', 'department', 'assets', 'licenses', 'accessories', 'consumables', 'createdBy',)
```

Comment:
This is a silly aesthetic thing, but it's weird that we don't normally use snakeCase and here is the one place where we do. But, that's the Laravel convention and we probably ought to follow it, so this is probably exactly how it should be.

Reply:
Duly ignored :P